src/playbackstrategy/msestrategy.js

```diff
@@ -700,7 +701,13 @@ function MSEStrategy(mediaSources, windowType, mediaKind, playbackElement, isUHD
         startAutoResumeTimeout()
       }
     },
-    play: () => mediaPlayer.play(),
+    play: () => {
+      if (windowType === WindowTypes.SLIDING) {
+        slidingWindowPausedTime = 0
```

Comment:
Is this needed?

If you only ever set it on an app pause then this will always result in a seek and be reset.

Reply:
This is just an artifact of an earlier attempt but I feel like it doesn't hurt to keep it in just to make sure the state is reset

Reply:
Whilst I agree in this instance it isn't that big a deal but don't feel it's good to keep around dead/redundant code.

Reply:
I'll remove it